README.md

```diff
@@ -30,7 +33,7 @@ The following sections are defined:
 | Key             | Description| Default
 |-----            |----------  | ---
 | extra_vars      | Path to YAML file containing Ansible extra-vars | extra_vars.yaml
-| fact_caching    | The [fact caching](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#cache-plugin) to use. <br> Set to _memory_ to revert to Ansible default ephemeral cache. | jsonfile 
+| fact_caching    | The [fact caching](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#cache-plugin) to use. <br> Set to _memory_ to revert to Ansible default ephemeral cache. | jsonfile
 |inventory        | Path to Ansible inventory | %(test_directory)s/inventory/hosts
 | iterations      | Number of iterations to run | 20
 | max_failures    | Number of failed iterations after which a test is disabled| 3
```

Comment:
Would be cool to mention that test_directory contains both functional and ha test folder